src/lab_sim/objectives/get_transform_frame_pose.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Same comment as above about adding to `picknik_ur_base_config`.

Reply:
Also, should this be a SubTree only Objective? I don't think you're going to be running this Objective on its own.

Reply:
> SubTree only Objective

I don't actually know what this means.
What should I do if I want this to be a behavior instead of a objective? Should I still add it to base_config?

Reply:
Hmm, this could exist in the base config I think, but maybe that should be a part of a wider effort where we move more Objectives into the base config, not just this one. Can you open up an issue for that?

Reply:
Are there a lot of objectives to move?

Reply:
I'm have no idea. There could be other Objectives in other configs that maybe general enough to move.

Reply:
https://github.com/PickNikRobotics/moveit_pro/issues/11134
https://github.com/PickNikRobotics/moveit_pro/issues/11135